pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -369,15 +368,17 @@ func (a Adapter) Deploy(parameters common.DeployParameters) (err error) {
 		knGvr := schema.GroupVersionResource{Group: "serving.knative.dev", Version: "v1", Resource: "routes"}
 		route, err := a.waitForManifestDeployCompletion(applicationName, knGvr, "Ready")
 		if err != nil {
+			s.End(false)
 			return errors.Wrap(err, "error while waiting for deployment completion")
 		}
 		fullURL = route.UnstructuredContent()["status"].(map[string]interface{})["url"].(string)
 	}
-	s.End(true)
 
 	if fullURL != "" {
-		log.Successf("URL for application %s: %s", applicationName, fullURL)
+		s.End(true)
+		log.Successf("Successfully deployed component: %s", fullURL)
 	} else {
+		s.End(false)
 		log.Errorf("URL unable to be determined for application %s", applicationName)
```

Comment:
As we are saying "Deploying component <componentName>" do we want to now chnage the error message to not mention <applicationName> and to use <componentName>

Reply:
yup